gulpfile.js/tasks/production.js

```diff
@@ -2,14 +2,20 @@ var gulp            = require('gulp')
 var gulpSequence    = require('gulp-sequence')
 var getEnabledTasks = require('../lib/getEnabledTasks')
 var os              = require('os')
+var fs              = require('fs')
 var path            = require('path')
 
 var productionTask = function(cb) {
   global.production = true
 
   // Build to a temporary directory, then move compiled files as a last step
   PATH_CONFIG.finalDest = PATH_CONFIG.dest
-  PATH_CONFIG.dest = path.join(os.tmpdir(), 'gulp-starter')
+  PATH_CONFIG.dest = PATH_CONFIG.temp
+      ? path.join(process.env.PWD, PATH_CONFIG.temp)
```

Comment:
hmm... wonder if we should change this to `path.join(process.env.PWD, PATH_CONFIG.temp, 'blendid')`, since the folder gets trashed every time. Wouldn't want someone accidentally pointing to a shared tmp folder, and have this task trash it, when other platforms were relying on it for caching or something.